tests/test_transforms.py

```diff
@@ -256,3 +256,11 @@ def test_eliminate_join_marks(self):
                 f"SELECT table1.id, table2.cloumn1, table3.id FROM table1 LEFT JOIN table2 ON table1.id = table2.id LEFT JOIN (SELECT tableInner1.id FROM tableInner1 LEFT JOIN tableInner2 ON tableInner1.id = tableInner2.id) {alias}table3 ON table1.id = table3.id",
                 dialect,
             )
+
+            # if multiple conditions, we check that after transformations the tree remains consistent
+            s = "select a.id from a, b where a.id = b.id (+) AND b.d (+) = const"
+            tree = parse_one(s, dialect=dialect)
+            assert ([type(t.parent_select) for t in tree.find_all(exp.Table)]) == [
+                exp.Select,
+                exp.Select,
+            ]
```

Comment:
This isn't really testing your change, right? You need to show how the AST is changed _after_ the transformation.